test/test_environment.py

```diff
@@ -100,7 +100,12 @@ def test_overridden_path(tmp_path, capfd):
     "build_frontend",
     [
         pytest.param("pip", marks=utils.skip_if_pyodide("No pip for pyodide")),
-        "build",
+        pytest.param(
+            "build",
+            marks=utils.skip_if_pyodide(
+                "pyodide doesn't support multiple values for PIP_CONSTRAINT"
```

Comment:
Do we have an issue for this?

Reply:
🤔 I don't think we do. @joerick might know about this, though!

I think it might have been `"build doesn't support multiple values for PIP_CONSTRAINT"`, rather than "pyodide".

Reply:
Here's my reference for that - https://github.com/pyodide/pyodide-build/blob/7126c5f1bd56ed7f40ce29cbcbe588267a42434c/pyodide_build/build_env.py#L339

Reply:
🤔 Thanks – I didn't remember that we had come across that issue previously. Would you like me to add that workaround to `pyodide-build`?

Reply:
That'd be great.

Reply:
So I was working at this for `pyodide-build` on a local branch, and I'd like to get some clarification here – did we want to allow multiple constraints files for `PIP_CONSTRAINT` for `pyodide-build`, or that `PIP_CONSTRAINT` should accept paths with spaces? I assume it should be the latter, as this is only for `cibuildwheel` as it combines constraints set by users with its own constraints?

Reply:
So I was working at this for `pyodide-build` on a local branch, and I'd like to get some clarification here – did we want to allow multiple constraints files for `PIP_CONSTRAINT` for `pyodide-build`, or that `PIP_CONSTRAINT` should accept paths with spaces? I assume it should be the latter, as this is only for `cibuildwheel` as it combines constraints set by users with its own constraints?

Reply:
Pip aren't gonna change this option for backwards-compatibility reasons, so pyodide-build is gonna have to work within Pip's behaviour here. 

That is, within the option value - spaces are _delimiters_ between files, and, if you want to pass a file with a space in the path, you have to use file URLs.

I guess, in your case, that will mean just passing through the option as-is, rather than checking for spaces? But I don't fully understand the reasoning for why `_create_constraints_file` looks like it does.

Reply:
Gotcha, thanks!